packages/core/test/acceptance/security_spec.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Component} from '@angular/core';
+import {TestBed} from '@angular/core/testing';
+
+
+describe('security', () => {
```

Comment:
```suggestion
describe('comment node text escaping', () => {
```
I would use a more specific name here since "security" is very broad

Reply:
done